fda-model/metrics/ConsequenceResults.cs

```diff
@@ -36,8 +36,6 @@ public bool IsNull
         public ConsequenceResults()
         {
             _consequenceResultList = new List<ConsequenceResult>();
-            ConsequenceResult dummyConsequenceResult = new ConsequenceResult();
```

Comment:
this was causing the "unassigned" functions

Reply:
Why was it done in the first place? It's a really strange move that probably should have been commented, but we didn't. I'm afraid to remove it now because I don't understand why it was there.